fastlane/Fastfile

```diff
@@ -15,15 +15,21 @@
 
 default_platform(:ios)
 
+BUILD_NUMBER =  if !ENV["CIRCLE_BUILD_NUMBER"].nil?
+                  ENV["CIRCLE_BUILD_NUMBER"]
+                else 
+                  ENV["BUILDKITE_BUILD_NUMBER"]
+                end
+
 platform :ios do
-  before_all do
-    setup_circle_ci
-  end
+  # before_all do
+  #   setup_circle_ci
+  # end
```

Comment:
We can remove this unused code. There is history in git if we ever want to bring it back.